internal/host/hostutil/test_utils.go

```diff
@@ -194,3 +241,28 @@ func GenerateInventoryWithResourcesWithBytes(cpu, memory int64, hostname string)
 	Expect(err).To(Not(HaveOccurred()))
 	return string(b)
 }
+
+func GenerateIPv4Addresses(count int, machineCIDR string) []string {
+
+	net1, _, err := net.ParseCIDR(machineCIDR)
+	Expect(err).NotTo(HaveOccurred())
+	net1Address := net1.To4()
+	ret := make([]string, count)
+	for i := 0; i < count; i++ {
+		net1Address[3]++
```

Comment:
This is incorrect way to increase IP - although it should be sufficient.
This is better https://github.com/openshift/assisted-installer-agent/blob/42d49b73098ae12d1879768f01c390fa952d1cae/src/free_addresses/free_addresses.go#L35

Reply:
I'll update the code. Thanks!